embuilder.py

```diff
@@ -201,6 +207,14 @@ def main():
       build(C_WITH_MALLOC, ['libdlmalloc_threadsafe_debug.bc'], ['-g', '-s', 'USE_PTHREADS=1', '-s', 'MALLOC="dlmalloc"'])
     elif what in ('dlmalloc_threadsafe', 'libc-mt', 'pthreads'):
       build(C_WITH_MALLOC, ['libc-mt.bc', 'libdlmalloc_threadsafe.bc', 'libpthreads.bc'], ['-s', 'USE_PTHREADS=1', '-s', 'MALLOC="dlmalloc"'])
+    elif what == 'dlmalloc_noerrno':
+      build(C_WITH_MALLOC, ['libdlmalloc_noerrno.bc'], ['-s', 'MALLOC="dlmalloc"', '-s', 'SUPPORT_ERRNO=0'])
+    elif what == 'dlmalloc_debug_noerrno':
+      build(C_WITH_MALLOC, ['libdlmalloc_debug_noerrno.bc'], ['-g', '-s', 'MALLOC="dlmalloc"', '-s', 'SUPPORT_ERRNO=0'])
+    elif what == 'dlmalloc_threadsafe_debug_noerrno':
+      build(C_WITH_MALLOC, ['libdlmalloc_threadsafe_debug_noerrno.bc'], ['-g', '-s', 'USE_PTHREADS=1', '-s', 'MALLOC="dlmalloc"', '-s', 'SUPPORT_ERRNO=0'])
+    elif what == 'dlmalloc_threadsafe_noerrno':
+      build(C_WITH_MALLOC, ['libdlmalloc_threadsafe_noerrno.bc'], ['-s', 'USE_PTHREADS=1', '-s', 'MALLOC="dlmalloc"', '-s', 'SUPPORT_ERRNO=0'])
```

Comment:
at some point we should make this programmatic, when the exponential blowup is bad enough ;) but fine for now, lgtm.